xml/System/ObsoleteAttribute.xml

```diff
@@ -54,43 +54,41 @@
-  
--   <xref:System.ObsoleteAttribute.IsError%2A>. This is a Boolean value that indicates to the compiler whether using the <xref:System.ObsoleteAttribute> attribute should cause it to emit an error (<xref:System.ObsoleteAttribute.IsError%2A> is `true`) or a warning (<xref:System.ObsoleteAttribute.IsError%2A> is `false`).  
-  
- For more information about using attributes, see [Attributes](/dotnet/standard/attributes/).  
-  
-## [!INCLUDE[win8_appname_long](~/includes/win8-appname-long-md.md)] Apps  
- When you create a Windows Metadata library (.winmd file), the <xref:System.ObsoleteAttribute> is exported as both the <xref:System.ObsoleteAttribute> attribute and the [Windows.Foundation.Metadata.DeprecatedAttribute](https://msdn.microsoft.com/library/windows/apps/windows.foundation.metadata.deprecatedattribute.aspx) attribute if only the <xref:System.ObsoleteAttribute> is present in source code. The <xref:System.ObsoleteAttribute> is transformed to the `DeprecatedAttribute` as follows:  
-  
--   If the `message` and `error` arguments are both present, `message` is assigned to the `DeprecatedAttribute` `message` argument. An error value of `true` maps to [DeprecationType.Remove](https://msdn.microsoft.com/library/windows/apps/windows.foundation.metadata.deprecationtype.aspx), and an `error` value of `false` maps to [DeprecationType.Deprecate](https://msdn.microsoft.com/library/windows/apps/windows.foundation.metadata.deprecationtype.aspx).  
-  
--   If the `message` argument is not supplied in the <xref:System.ObsoleteAttribute>, its default value in the `DeprecatedAttribute` is "*element_name* is deprecated", where *element_name* is the name of the target program element to which the attribute is applied.  
-  
--   If the `error` argument is not present in the <xref:System.ObsoleteAttribute>, its default value in the `DeprecatedAttribute` is [DeprecationType.Deprecate](https://msdn.microsoft.com/library/windows/apps/windows.foundation.metadata.deprecationtype.aspx).  
-  
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+<xref:System.ObsoleteAttribute> is applicable to all program elements except assemblies, modules, parameters, and return values. Marking an element as obsolete informs users that the element may be removed in a future version of the product.
+
+The string assigned to the <xref:System.ObsoleteAttribute.Message%2A> property is emitted by the compiler when the attribute target is used in code. Ideally, the string should provide some workaround or programmatic alternative.
+
+Use the <xref:System.ObsoleteAttribute.IsError%2A> property to indicate to the compiler whether using the <xref:System.ObsoleteAttribute> attribute should cause it to emit an error (<xref:System.ObsoleteAttribute.IsError%2A> is `true`) or a warning (<xref:System.ObsoleteAttribute.IsError%2A> is `false`).
+
+For more information about using attributes, see [Attributes](/dotnet/standard/attributes/).
+
+## Windows 8.x Store apps
```

Comment:
Do we still need this section?

Reply:
> ## Windows 8.x Store apps
> Do we still need this section?

I'm not sure.  @terrajobst, what do you think?